src/scss/_typography_theme_colors.scss

```diff
@@ -0,0 +1,56 @@
+/* internal theme text styles
+ * kept in a separate scss file for easier use in components
+ * ex: import this file in a vue component to be able to @apply these classes
+ */
+
+/* Deprecation warning:
+ * .text-theme-red and .text-theme-red-hover are deprecated and will be removed in the next major release.
+ * Use .text-theme-color and .text-theme-color-hover instead.
+ */
+@layer utilities {
+  @responsive {
+    @variants hover {
+      .text-theme-color,
+      .text-theme-red {
+        @apply text-jpl-red;
+      }
+
+      .text-theme-color-hover,
+      .text-theme-red-hover {
+        @apply text-jpl-red-dark;
+      }
+    }
+
+    .ThemeLight {
+      .text-theme-color,
+      .text-theme-red,
+      .hover\:text-theme-color:hover,
+      .hover\:text-theme-red:hover {
```

Comment:
Tailwind's `@variant hover` is heavy handed with nested classes and would generate things like the following (even if the `@variant` directive was nested within the parent class):

```
.hover\ThemeLight:hover .hover\:text-theme-color:hover
```

The only workaround I could figure out was to hand write the nested hover variants.